dev-haskell/haddock-api/haddock-api-2.24.0.ebuild

```diff
@@ -16,6 +16,8 @@ SLOT="0/${PV}"
 KEYWORDS="~amd64 ~x86 ~amd64-linux ~x86-linux"
 IUSE=""
 
+RESTRICT="test" # due to <dev-haskell/quickcheck-2.14
```

Comment:
Is it a runtime test failure or build time?

Reply:
Build time. I figured itsde sense to RESTRICT since there will be slot conflicts otherwise.

I can try all these with quickcheck 2.14, however that will have to wait until after it is unmasked which needs to wait for aeson

Reply:
`RESTRICT=test` does not avoid build-time dependencies. It only avoids running tests. `USE=test emerge -v1 ...` will still pull in test dependencies.